translator/util/sdkutil.go

```diff
@@ -80,6 +80,16 @@ func DetectKubernetesMode(configuredMode string) string {
 
 }
 
+func DetectWorkloadType() string {
```

Comment:
If we are returning a value, we should name the func "Get*"

also if we are checking for an error, maybe it's worth returning an error as well?

```suggestion
func GetWorkloadType() (string, error) {
```

Reply:
We don't want to return an error (or warning) here since if it defaults to `""`, then it just means they are not on Kubernetes, which is fine.